shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rewrite/condition/EncryptConditionEngine.java

```diff
@@ -131,6 +128,17 @@
         return Optional.empty();
     }
     
+    private Optional<EncryptCondition> createEncryptCondition(final BinaryOperationExpression expression, final String tableName) {
+        String operator = expression.getOperator();
+        if (!LOGICAL_OPERATOR.contains(operator)) {
+            if (isSupportedOperator(operator)) {
+                return createCompareEncryptCondition(tableName, expression, operator, expression.getRight());
+            }
+            throw new ShardingSphereException(String.format("The SQL clause '%s' is unsupported in encrypt rule.", operator));
```

Comment:
@cheese8 Please remove String.format here, ShardingSphereException has include this logic.

Reply:
ok